src/main/osd/osd_elements.c

```diff
@@ -599,7 +603,54 @@ static void osdElementArtificialHorizon(osdElementParms_t *element)
         }
     }
 
-    element->drawElement = false;  // element already drawn
+   float earthUpinBodyFrame[3] = {-rMat[2][0], -rMat[2][1], -rMat[2][2]}; //transforum the up vector to the body frame
+   int thetaB; // pitch from body frame to up vector
+   int psiBSign; // psi from body frame to up vector
+   char arrowDir; // direction to draw arrow
+   float arrowDirFloat;
+   if(attitude.values.pitch>0.0){ //nose down
+       thetaB = asinf(earthUpinBodyFrame[2]) * (1800.0f / M_PIf); // get theta
```

Comment:
Using simple projection on (camera) plane may be enough.  Displaying zenit/nadir when it's far from camera direction isn't that useful (and there are other visual clues  on OSD in that case).